src/presenters/addressPresenter.js

```diff
@@ -10,9 +10,9 @@ module.exports = {
 
     const fullLocality = localityNames.join(" ");
     if (fullStreetName) {
-      return `${fullBuildingName} ${fullStreetName}, ${fullLocality}, ${address.postalCode}`;
+      return `${fullBuildingName} ${fullStreetName}, ${fullLocality}, ${address.postalCode}`.trim();
```

Comment:
@MattBidewell you were asking about `trim()` here. In some cases `fullBuildingName` may be `""`, and this would result in a leading space